client/stylesheets/_toCat.scss

```diff
@@ -3788,12 +3793,15 @@ On click the images will be displayed at normal size to complete the effect
     border-radius: 6px;
 }
 
-
+.profileUserNameHidden, .profileUserGraphHidden {
+    display: none !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces